tests/lsp_features/completion.zig

```diff
@@ -1756,6 +1808,12 @@ fn testCompletion(source: []const u8, expected_completions: []const Completion)
             return error.InvalidCompletionKind;
         }
 
+        if (expected_completion.documentation != null and actual_completion.documentation != null) {
```

Comment:
the test should fail when `expected_completion.documentation != null and actual_completion.documentation == null`